<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZShareDialog</class>
 <widget class="QDialog" name="ZShareDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dmix/dsnoop/dshare plugin properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QGroupBox" name="groupIPC">
       <property name="title">
        <string>IPC access</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Unique IPC key</string>
            </property>
            <property name="buddy">
             <cstring>comboIPCKey</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboIPCKey">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Permissions</string>
            </property>
            <property name="buddy">
             <cstring>comboIPCPerm</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboIPCPerm">
            <property name="editable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupParams">
       <property name="title">
        <string>Parameters</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>HW ptr alignment</string>
            </property>
            <property name="buddy">
             <cstring>comboHWPtrAlignment</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboHWPtrAlignment"/>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCheckBox" name="checkSlowPtr">
          <property name="toolTip">
           <string>Slow but more precise pointer updates</string>
          </property>
          <property name="text">
           <string>Slow ptr</string>
          </property>
          <property name="tristate">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBindings">
     <property name="title">
      <string>Bindings</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QTableWidget" name="tableBindings"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="buttonAddBinding">
          <property name="text">
           <string>Add</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDeleteBinding">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboIPCKey</tabstop>
  <tabstop>comboIPCPerm</tabstop>
  <tabstop>comboHWPtrAlignment</tabstop>
  <tabstop>checkSlowPtr</tabstop>
  <tabstop>tableBindings</tabstop>
  <tabstop>buttonAddBinding</tabstop>
  <tabstop>buttonDeleteBinding</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ZShareDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>385</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ZShareDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>391</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
